tests/Crowdin.Api.UnitTesting/Tests/StringCorrections/StringCorrectionsApiTests.cs

```diff
@@ -0,0 +1,263 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Crowdin.Api.Core;
+using Crowdin.Api.StringCorrections;
+using Crowdin.Api.StringTranslations;
+using Moq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Crowdin.Api.UnitTesting.Tests.StringCorrections;
+
+public class StringCorrectionsApiTests
+{
+    private static readonly JsonSerializerSettings DefaultSettings = TestUtils.CreateJsonSerializerOptions();
+
+    [Fact]
+    public async Task ListCorrections_ShouldReturnCorrections()
+    {
+        const int projectId = 1;
+        const int stringId = 2;
+        const int limit = 25;
+        const int offset = 0;
+
+        var corrections = new[]
+        {
+            new Correction
+            {
+                Id = 1,
+                Text = "This string has been corrected",
+                PluralCategoryName = "few",
+                User = new User
+                {
+                    Id = 5,
+                    Username = "john_doe",
+                    FullName = "John Doe",
+                    AvatarUrl = "https://example.com/avatar.png"
+                },
+                CreatedAt = DateTimeOffset.Parse("2023-10-01T12:00:00Z"),
+            }
+        };
+
+        JObject responseObject = JObject.FromObject(new
```

Comment:
We use RESX resources for request and response JSONs mocking. Please consider the same approach and check how other API sections are implemented